CONTRIBUTING.md

```diff
@@ -57,19 +57,32 @@ _For this example, we will use the tag `v0.1.0`. This tag already exists, so you
 
 Create a proper SemVer compatible tag. Consult the [SemVer specification](https://semver.org/spec/v2.0.0.html) if you are unsure what this means.
 
-`git tag v0.1.0`
+```bash
+git tag v0.1.0
+```
 
 **Step 2**
 
 Push the tag GitHub.
 
-`git push origin v0.1.0`
+```bash
+git push origin v0.1.0
+```
 
 This command will trigger the [Release GitHub Action](https://github.com/posit-dev/posit-sdk-py/actions/workflows/release.yaml).
 
+Once complete, the release will be available on [PyPI](https://pypi.org/project/posit-sdk).
+
 **Step 3**
 
-Once complete, the release will be available on [PyPI](https://pypi.org/project/posit-sdk).
+Create a release on GitHub. Please following the pattern established in previous releases. Set the title to the tag name (e.g., `v0.1.0`) and the body to the generated release notes. For reference, see <https://github.com/posit-dev/posit-sdk-py/releases>.
```

Comment:
Should we also add a note about enabling a discussion for the release?

Reply:
Yep! Thank you!